front/src/modules/ui/input/components/IconPicker.tsx

```diff
@@ -0,0 +1,80 @@
+import { useMemo, useState } from 'react';
+import styled from '@emotion/styled';
+
+import { LightIconButton } from '@/ui/button/components/LightIconButton';
+import { DropdownMenuSearchInput } from '@/ui/dropdown/components/DropdownMenuSearchInput';
+import { StyledDropdownMenu } from '@/ui/dropdown/components/StyledDropdownMenu';
+import { StyledDropdownMenuItemsContainer } from '@/ui/dropdown/components/StyledDropdownMenuItemsContainer';
+import { StyledDropdownMenuSeparator } from '@/ui/dropdown/components/StyledDropdownMenuSeparator';
+import { IconComponent } from '@/ui/icon/types/IconComponent';
+
+type IconPickerProps = {
+  icons: Record<string, IconComponent>;
```

Comment:
@thaisguigon let's merge it but I think we should not pass the icons to the iconPicker, it should be part of it.
the icon picker should take an "icon" url or an iconList (enum = 'tabler-icons' only for now) and load them asynchronously internally
This can be treated in another PR if you prefer!

Reply:
Ok, we might want to pass the icon key + the icon component to `onChange` then.
This is a first, generic implementation, I was thinking we could discuss the icon loading method later and implement it in another PR/issue.

Reply:
ok!